reading.Rmd

```diff
@@ -1,3 +1,8 @@
+---
+output:
+  pdf_document: default
```

Comment:
What's this stuff?

Reply:
Ohhh, let's remove that. I must have got put there when I click "knit" to test the note/warning stuff above. I didn't notice RStudio did that...

Reply:
removed